cases/models.py

```diff
@@ -258,6 +258,12 @@ def update_location_cache(self):
                     desc = f"({self.point.x:.0f},{self.point.y:.0f})"
             self.location_cache = f"{self.radius}m around {desc}"
 
+        if self.estate:
+            pass
+        elif self.point:
+            estate = cobrand.api.in_an_estate(self.point)
+            self.estate = "y" if estate else "n"
+
```

Comment:
I did wonder this, but it is also part of the location, and `update_location_cache` is called in a few places assuming it will update the necessary. This also needs to come after in case the first part of the function has looked up the point, so can't really be called independently.